src/posts/2021-09-08-making-links-accessible.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Making links accessible
+description: Tips on how to make links accessible.
+category: How to
+author: Pascal Akunne
+date: 2021-09-08
+further_reading:
+  - title: Links and Hypertext
+    url: https://webaim.org/techniques/hypertext/
+    source:  WAIM Web Accessibility In Mind
+    year: 2021
+  - title: Link Purpose (Link only)
+    url: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html
+    source:  WCAG 2.1
+    year: 2016
+  - title: Links
+    url: https://accessibility.umn.edu/what-you-can-do/start-7-core-skills/links
+    source: University of Minnesota
+    year: 2021
+tags:
+  - howto
+---
+
+Links are one of the primary elements in web development. Links can be made accessible by making them clear, descriptive and purposeful out of context. This helps break accessibility barriers for both people with visual disability and clear sighted, as well as people with cognitive disability. Links should be made obvious so people can understand what is means. By default, links are underlined by the browser. 
+
+
+## Guidelines for making links accessible
+
+### Don'ts
+
+- Careless use of ambiguous link texts like `Click here`, `More...`, `Link` are not appropriate and should be avoided. Rather, a text that describes the purpose of that link should be used. 
```

Comment:
don't need the word "careless" as that makes an unnecessary negative assumption of the person writing the text.

additionally "are not appropriate and should be avoided" also lacks context.  For example "more options", "learn more", when used in the context of a paragraph are acceptable to meet 2.4.4.  If you are going to say they're inappropriate, you need to provide context as to why.